modules/packages/Sort.chpl

```diff
@@ -459,19 +460,40 @@ algorithm used is made by the implementation.
 :type Data: [] `eltType`
 :arg comparator: :ref:`Comparator <comparators>` record that defines how the
   data is sorted.
-
+:arg stable: Defaults to ``false``. If it is ``false``, the implementation
+  can sort in a way that reorders equal keys. If it is ``true``, it will use a
+  stable algorithm in order to preserve the order of equal keys.
+:arg mimimizeMemory: Defaults to ``false``. If it is ``false``, the
+  implementation can make a copy of ``Data`` for scratch storage during the
+  sort. If it is ``true``, it will use an in-place algorithm in order to use
+  less memory.
```

Comment:
I suggest calling it `inPlace` because it is a better counterpart to `stable` - shorter and more conventional.

Please add a warning to this chpldoc that `stable=true` is currently not implemented.

I suggest adding a nod to the available distributed-sort functionality.

Reply:
I thought about `inPlace` but my concern is that it conflicts with a different meaning. If we say "in place algorithm", it is clear. But if I say "`sort` is in-place", I would expect that to simply mean that `sort(A)` stores the result of sorting in `A`. Vs if `sort` were not "in-place", I would expect that `sort(A)` would return a new array that was sorted, without modifying `A`. This is the reason that I did not use this term here. I think that `minimizeMemory` is clearer in the context of the `sort` call, and that using "in-place algorithm" in the description addresses the need to connect it to terminology that people might be familiar with.

Reply:
(Of course, another option would be to name the formal `inPlaceAlgorithm`, which would address my concern)

Reply:
I see, it is like `sort` vs. `sorted` in Python. Looking back at minimizeMemory, it makes me want to say `minimizeMemoryUse` instead. If it does not work, then I like `inPlaceAlgorithm` too.

Reply:
I will go with `inPlaceAlgorithm` here.